crypto/riscvcap.c

```diff
@@ -71,6 +77,38 @@ static void parse_env(const char *envstr)
     }
 }
 
+#ifdef OSSL_RISCV_HWPROBE
+static long __riscv_hwprobe(struct riscv_hwprobe *pairs, size_t pair_count,
```

Comment:
Do not use double underscore prefix on identifiers. This is static so imo `riscv_hwprobe` is fine.